SingularityBase/src/main/java/com/hubspot/singularity/SingularityPendingDeploy.java

```diff
@@ -10,14 +10,16 @@
   private final Optional<SingularityLoadBalancerUpdate> lastLoadBalancerUpdate;
   private final DeployState currentDeployState;
   private final Optional<SingularityDeployProgress> deployProgress;
+  private final Optional<SingularityRequest> newRequestData;
```

Comment:
naming seems a little awkward in my opinion -- `updatedRequest` might make more sense